frontend/src/v5/store/tickets/card/ticketsCard.selectors.ts

```diff
@@ -20,11 +20,11 @@ import { SequencingProperties, TicketBaseKeys, TicketsCardViews } from '@/v5/ui/
 import { createSelector } from 'reselect';
 import { selectTemplateById, selectTemplates, selectTicketById, selectTickets } from '../tickets.selectors';
```

Comment:
it's a general problem with `oneOf` `manyOf` where we should've checked for duplicates at validation. (created an issue for it too : https://github.com/3drepo/3drepo.io/issues/4796)

Don't worry about it, my aim is to fix it before we release... so we can assume this can never happen.